pages/common/views/components/form.jsx

```diff
@@ -9,6 +9,7 @@ import ConditionalReveal from './conditional-reveal';
 const fields = {
   inputText: props => <Input { ...props } />,
   inputEmail: props => <Input type="email" { ...props } />,
+  inputPassword: props => <Input type="password" { ...props } />,
```

Comment:
I feel like there's scope here for a `type` prop and then everything is `<Input/>`. Save inventing a new top-level object every time we want a different `type` value.

This works fine too for now though.

Reply:
yeah, agree!